realm-annotations-processor/src/test/resources/io/realm/RealmDefaultModule.java

```diff
@@ -0,0 +1,7 @@
+package io.realm;
```

Comment:
Autogenerated files doesn't have headers. Adding the header would break our unit tests.